index.html

```diff
@@ -1893,8 +1900,16 @@ <h4>
       <h2>
         Accessibility Considerations
       </h2>
-      <p class="issue" title="Work in Progress">
-        This section is a work in progress as this document evolves.
+      <p>
+        Credential presentation and issuance facilitated by the Digital Credentials API have many
+        of the same accessibility considerations as detailed in [[[credman-accessibility]]] of the
+        Credential Management API.
+      </p>
```

Comment:
The problem is that Cred Man doesn't detail any "accessibility considerations" (right now). So if someone goes looking, they would be disappointed.

```suggestion
```

I suggest we remove this, until such time as Cred Man actually gets accessibility considerations.

Reply:
Makes sense, I've removed this ✌️